Books/client/src/App.test.tsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
 import { render, screen } from '@testing-library/react';
+import React from 'react';
```

Comment:
Should we have a test for App right now?

Reply:
I don't know too much about testing, what do you mean?

Reply:
I mean, there's not much to test for in App.tsx so maybe you could delete this test and its import in App.tsx

Reply:
Done, deleted it for now

Reply:
Great!